src/viewProviders/baseModels/base.ts

```diff
@@ -230,144 +227,3 @@ export abstract class BaseViewProvider<T extends BaseViewProviderData>
     return await window.withProgress(progressOptions, task);
   }
 }
-
-/**
- * Type describing 'focused parent' types for ParentedBaseViewProvider,
- * namely things which either are or come from a single Environment.
- */
-type EnvironmentedBaseViewProviderData = BaseViewProviderData & {
-  environmentId: EnvironmentId;
-};
-
-/**
- * Base class for all tree view providers handling a primary resource type and a parent resource.
- * @template P The type of the "parent" resource that can be "focused" in the view to determine which
- * resources will be shown. (Example: `KafkaCluster`, `SchemaRegistry`, `FlinkComputePool`)
- * @template T The primary resource(s) that will be shown in the view.
- */
-export abstract class ParentedBaseViewProvider<
-    P extends EnvironmentedBaseViewProviderData,
-    T extends BaseViewProviderData,
-  >
```

Comment:
All of this was moved over to [`src/viewProviders/baseModels/parentedBase.ts`](https://github.com/confluentinc/vscode/blob/b82709411a49bd78153297531935ee562c53d6d2/src/viewProviders/baseModels/parentedBase.ts)